src/api/v1/endpoints/plans.py

```
@@ -0,0 +1,113 @@
+    )
+    db.add(plan)
+    db.commit()
+    db.refresh(plan)
+    return SuccessResponse(data=SavedPlanResponse.model_validate(plan))
+
+
+@router.get("/", response_model=SuccessResponse[list[SavedPlanResponse]])
+def get_plans(
+    year: int = Query(...),
+    month: int = Query(...),
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)